### ISSUE TITLE
How to manually test CIFAR10 convnet example?

### ISSUE THREAD

**Issue by robben1234** [tflearn/tflearn#448]
Hello. I'm trying to get familiar with TFLearn and ML in general. I have used [example code](https://github.com/tflearn/tflearn/blob/master/examples/images/convnet_cifar10.py) for convnet CIFAR10 to train model and now i can't understand how to manually test it like in [tutorial with Titanic](https://github.com/tflearn/tflearn/blob/master/tutorials/intro/quickstart.md). I wan't to write something like this:

```
img = open('pic.jpg', 'r')
model = tflearn.DNN(network)
model.load("convnetcifar10.tfl")

pred = model.predict(image)
print("Results:", pred[0][1])
```

but don't know how to prepare image to insert it into `model.predict`

**Comment by robben1234** [comment]
I figured out this problem. Here's code:
```
model = tflearn.DNN(network)
model.load("convnetcifar10.tfl")

from PIL import Image, ImageOps
import numpy as np
img = Image.open(img_file).convert('RGB')
img = ImageOps.fit(img, ((32,32)), Image.ANTIALIAS)

img_arr = np.array(img)
img_arr = img_arr.reshape(1,32,32,3).astype("float")

pred = model.predict(img_arr )
print("Results: %s"%pred[0])
```

Though, model that showed ~87% accuracy while training, now telling me that 2/33 pics of cats is actually are airplanes. `pred` has always have 1 dimension, so i can't be reading wrong data from prediction. So i think problem only could be in wrong trained model?

**Comment by aymericdamien** [comment]
87% training is it training or testing accuracy? Your model is maybe overfitting.

**Comment by robben1234** [comment]
After 50 epochs: accuracy 0.8842 (raw 0.8854), validation 0.8151. 

![screenshot 2016-11-10 04 18 07](https://cloud.githubusercontent.com/assets/5873726/20162308/c354b1aa-a6fc-11e6-86af-f7540d47aa75.png)

Model uses code from [tflearn/examples](https://github.com/tflearn/tflearn/blob/master/examples/images/convnet_cifar10.py). I'm not sure it needs improvements.

```
network = input_data(shape=[None, 32, 32, 3],
                     data_preprocessing=img_prep,
                     data_augmentation=img_aug)
network = conv_2d(network, 32, 3, activation='relu')
network = max_pool_2d(network, 2)
network = conv_2d(network, 64, 3, activation='relu')
network = conv_2d(network, 64, 3, activation='relu')
network = max_pool_2d(network, 2)
network = fully_connected(network, 512, activation='relu')
network = dropout(network, 0.5)
network = fully_connected(network, 10, activation='softmax')
network = regression(network, optimizer='adam',
                     loss='categorical_crossentropy',
                     learning_rate=0.001)
```

Also, i have used more images (around 50 cats) to test model and it proven ~70% accuracy. So maybe first time problem was in cats.
